setup.py

```diff
@@ -116,6 +116,7 @@ def run(self):
     "MySQL-python>=1.2.5",
     "numpy>=1.7",
     "nydus>=0.10.8",
+    "oauthlib==0.7.2",
```

Comment:
Can you add `FIXME` or more verbose commit message with a reason why we pin the package version and what should be done to unpin it again. Thanks